app/views/devise/registrations/new.html.erb

```diff
@@ -21,6 +31,26 @@
     <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
   </div>
 
+<div class="field">
+    <%= f.label :address %><br />
+    <%= f.text_field :address, autofocus: true, autocomplete: "Address" %>
```

Comment:
Did you mean to use `placeholder` vs `autocomplete`?  The others below also have this.

I noticed that `autofocus` was turned on for every `text_field`, maybe `autofocus` should only be on the first field?

Reply:
Yes, you were totally right on both accounts. Thanks!